pkg/scheduler/core/generic_scheduler.go

```diff
@@ -83,7 +83,7 @@ func (g *genericScheduler) Schedule(ctx context.Context, placement *policyv1alph
 	}
 	klog.V(4).Infof("selected clusters: %v", clusters)
 
-	clustersWithReplicas, err := g.assignReplicas(clusters, placement.ReplicaScheduling, spec)
+	clustersWithReplicas, err := g.assignReplicas(ctx, clusters, placement.ReplicaScheduling, spec)
```

Comment:
The `ctx` is unused, not at least by this PR. 
Just a guess, it might be used in the future, but please postpone it. Because I'm not sure we can finish all the optimized work in this release, an unused parameter might bring annoying static check warnings.